assets/app/view/game/stock_market.rb

```diff
@@ -145,7 +145,7 @@ def grid_2d
         @game.stock_market.market.flat_map do |row_prices|
           row = row_prices.map do |price|
             if price
-              corporations = price.corporations
+              corporations = price.corporations.reject(&:closed?)
```

Comment:
if a corporation is closed they shouldn't be in the price anyways, so i don't know if we should do this

Reply:
I added a close(corporation) method in stock_market that removes it from price. That works.